external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -986,7 +1003,7 @@ pub fn get_symbols(
     let mut parsed_ast = None;
     let mut typed_ast = None;
     let mut diagnostics = None;
-    build_plan.compile_with_driver(&mut std::io::sink(), |compiler| {
+    build_plan.compile_with_driver(Some(Box::new(vfs)), &mut std::io::sink(), |compiler| {
```

Comment:
Concretely what I'm suggesting 
```suggestion
    build_plan.compile_with_driver(&mut std::io::sink(), |compiler| {
        let compiler = compiler.set_file_reader(Box::new(vfs));
```

Reply:
I understood what the suggestion was, particularly that I tried this myself. The problem here (at least as I understand it) is that Rust needs to copy `vfs` into the closure, which would involve making a copy of all source files read from the file system before `compile_with_driver` is invoked. I am trying to avoid it by passing `vfs` by-value to the function itself.